src/storage/mutate/DeleteVerticesProcessor.cpp

```diff
@@ -0,0 +1,147 @@
+            auto part = pv->first;
+            const auto& vertices = pv->second;
+            for (auto v = vertices.begin(); v != vertices.end(); v++) {
+                auto prefix = NebulaKeyUtils::vertexPrefix(part, *v);
+
+                // Evict vertices from cache
+                if (FLAGS_enable_vertex_cache && vertexCache_ != nullptr) {
+                    std::unique_ptr<kvstore::KVIterator> iter;
+                    auto ret = this->kvstore_->prefix(spaceId, part, prefix, &iter);
+                    if (ret != kvstore::ResultCode::SUCCEEDED) {
+                        VLOG(3) << "Error! ret = " << static_cast<int32_t>(ret)
+                                << ", spaceID " << spaceId;
+                        this->onFinished();
+                        return;
+                    }
+
+                    while (iter->valid()) {
+                        auto key = iter->key();
+                        if (NebulaKeyUtils::isVertex(key)) {
+                            auto tag = NebulaKeyUtils::getTagId(key);
+                            VLOG(3) << "Evict vertex cache for VID " << *v << ", TagID " << tag;
+                            vertexCache_->evict(std::make_pair(*v, tag), part);
+                        }
+                        iter->next();
+                    }
+                }
+                doRemovePrefix(spaceId, part, std::move(prefix));
+            }
+        }
+    } else {
+        callingNum_ = req.parts.size();
+        std::for_each(req.parts.begin(), req.parts.end(), [&](auto &partVerticse) {
+            auto partId = partVerticse.first;
+            const auto &vertices = partVerticse.second;
+            auto atomic = [&]() -> std::string {
+                return deleteVertices(spaceId, partId, vertices);
+            };
+
+            auto callback = [spaceId, partId, this](kvstore::ResultCode code) {
+                VLOG(3) << "partId:" << partId << ", code:" << static_cast<int32_t>(code);
+                handleAsync(spaceId, partId, code);
+            };
+            this->kvstore_->asyncAtomicOp(spaceId, partId, atomic, callback);
+        });
+    }
+}
+
+std::string DeleteVerticesProcessor::deleteVertices(GraphSpaceID spaceId,
+                                                    PartitionID partId,
+                                                    const std::vector<VertexID>& vertices) {
+    std::unique_ptr<kvstore::BatchHolder> batchHolder = std::make_unique<kvstore::BatchHolder>();
+    for (auto& vertex : vertices) {
+        auto prefix = NebulaKeyUtils::vertexPrefix(partId, vertex);
+        std::unique_ptr<kvstore::KVIterator> iter;
+        auto ret = this->kvstore_->prefix(spaceId, partId, prefix, &iter);
+        if (ret != kvstore::ResultCode::SUCCEEDED) {
+            VLOG(3) << "Error! ret = " << static_cast<int32_t>(ret)
+                    << ", spaceId " << spaceId;
+            return "";
+        }
+        TagID latestVVId = -1;
```

Comment:
`latestVVId` Can’t understand this abbreviation..

Reply:
It's seems use to avoid duplicate version